docs/integrations/microsoft-azure/azure-load-balancer.md

```diff
@@ -20,24 +22,177 @@ For more information on supported metrics, refer to [Azure documentation](https:
 
 ## Setup
 
+:::note
+This app supports only load balancers of type **Standard** and **Gateway**.
```

Comment:
```suggestion
This app supports only load balancers of **Standard** and **Gateway** SKU.
```

Reply:
Type is categorized as  Public and Internal hence using the SKU keyword 
<img width="636" alt="image" src="https://github.com/user-attachments/assets/4eb764d4-c9a6-4058-8961-5730964eb67c">